### PR TITLE
Update @alerts.components adding province

### PR DESCRIPTION
**Pull request by cloudcongo**
Adding province field to src/app/pages/alerts/alerts.component.html and other minor changes.

Author: Adi from @floss.cd Whatsapp group

### CODE REVIEW

**Comment by bernard-ng** [comment]
@cloudcongo the built travis failed, please correct linting errors

**Comment by bernard-ng** [comment]
@cloudcongo  to add the province the backend has to be prepared to receive it, adding it in the frontend without going through the backend is a waste of time, if possible add the feature to the backend first, or open an issue for the feature to be added to the backend before.

**Comment by bernard-ng** [comment]
sorry @adi, there was a small mistake when mentioning another user

**Comment by bernard-ng** [comment]
@cloudcongo  run ```$ ng lint```  to fix linting error

File: src/app/components/header/header.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
       <div class="flex items-center justify-between">
         <div class="flex items-center">
           <h2 class="text-xl font-bold leading-tight text-gray-900">
-            Covid19 {{'Info' | translate }}
+            {{'Apercu' | translate }}
```
Comment (bernard-ng) [inline src/app/components/header/header.component.html]: fix typo  : "Aperçu"

Comment (cloudcongo) [reply src/app/components/header/header.component.html]: @bernard-ng ok - I will get it fixed.

**Pull request closed without merging**